wsd/COOLWSD.cpp

```diff
@@ -6383,7 +6356,9 @@ void forwardSigUsr2()
 int main(int argc, char** argv)
 {
     SigUtil::setUserSignals();
-    SigUtil::setFatalSignals("wsd " COOLWSD_VERSION " " COOLWSD_VERSION_HASH);
+    if (!Util::isKitInProcess()) {
+        SigUtil::setFatalSignals("wsd " COOLWSD_VERSION " " COOLWSD_VERSION_HASH);
+    }
```

Comment:
Why? Why remove the fatal signal in 'kit in process' mode?

Reply:
ohh yes, sorry we only need to remove fatal signal for Forkit and Kit not for COOLWSD ( pushing that change...)